src/components/molecules/ZeroResults/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react";
+import { ZeroResultsProps } from "./interface";
+
+import * as S from "./styles";
+
+const ZeroResults: FC<ZeroResultsProps> = ({ setTag, setSearchTerm }) => {
+  const suggestions = ["Ciągi", "Jedzenie", "Zwierzęta"];
+
+  const handleOnClick = (el: string) => {
+    setTag(el);
+    setSearchTerm(el);
+  };
+
+  return (
+    <S.ZeroResultsWrapper>
+      <S.ZeroResultsImg aria-label="Dziewczyna patrząca przez lornetkę" />
```

Comment:
I'd skip `aria-label` and would actually hide this decoration element from screen reader ;)
```suggestion
      <S.ZeroResultsImg aria-hidden="true" />
```

Reply:
@bazelajoanna

Reply:
Done ;)